packages/support/src/settings/_index.scss

```diff
@@ -1,22 +1,14 @@
-// Vendor overrides
-@import 'override.uswds';
-
-// Vendor variables
-// Defined after overrides so overrides work
-
 // Manually set Bourbon's font stack variables. This allows us to avoid
 // installing Bourbon as a USWDS dependency.
 $georgia: 'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
```

Comment:
I don't see these font variables used anywhere, but I left them in just in case

Reply:
I don't see them used either and am thinking we should remove them